spec/tweet_generator_spec.rb

```diff
@@ -1,15 +1,15 @@
-require 'cute_pets/tweet_generator'
+require 'cute_pets/post_generator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.